### DISCUSSION TITLE
NextAuth returns expired Okta access token?

### DISCUSSION THREAD

**Discussion by brandonchinn178** [nextauthjs/next-auth#1352] [Help] [answered]
It seems like the NextAuth token expires after 30 days, but Okta access tokens expire after 1 hour (by default), so NextAuth returns a session with an expired `accessToken` that I can't use anymore. Is there a way to get NextAuth to refresh the token?

**Comment by balazsorban44** [comment, marked as answer]
https://next-auth.js.org/tutorials/refresh-token-rotation
